assets/prototype/domain/shared/entities/tickets/predicates/sorters/index.test.ts

```diff
@@ -0,0 +1,72 @@
+import { compareAsc, parseISO } from 'date-fns';
+
+import sorters from './index';
+import { SortTickets } from '../../../../../eventEditor/data/ticket/types';
+import { nodes as tickets } from '../../../../../eventEditor/data/queries/tickets/test/data';
+
+describe('sorters', () => {
+	it('should return tickets sorted in default chronological order if no order prop is provided', () => {
+		const sortedTickets = sorters({ tickets });
+		sortedTickets.forEach((currentTicket, index) => {
+			const nextTicket = index + 1 <= sortedTickets.length ? sortedTickets[index + 1] : null;
+			if (nextTicket) {
+				const chronologicComparison = compareAsc(
+					parseISO(currentTicket.startDate),
+					parseISO(nextTicket.startDate)
+				);
+				expect(chronologicComparison).toBe(-1);
+				expect(currentTicket.id < nextTicket.id).toBe(-1);
+				expect(currentTicket.name < nextTicket.name).toBe(-1);
```

Comment:
hmmm will this **always** be the case?
it means that anyone adding additional tickets to the test data will have to follow that same pattern and ensure that the IDs and names are sorted the same as the dates!
might be wise to just remove this, since it's not relevant to any EE business rules, and has the potential to cause unnecessary fails if test data is modified

Reply:
removed those 2 line :+1: